apps/mobile/src/screens/Onboarding/OnboardingRecommendedUsers.tsx

```diff
@@ -0,0 +1,201 @@
+import { useNavigation } from '@react-navigation/native';
+import { Suspense, useCallback, useMemo, useState } from 'react';
+import { useWindowDimensions, View } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { graphql, useLazyLoadQuery, usePaginationFragment } from 'react-relay';
+import { RightArrowIcon } from 'src/icons/RightArrowIcon';
+
+import { BackButton } from '~/components/BackButton';
+import { Button } from '~/components/Button';
+import { GalleryTouchableOpacity } from '~/components/GalleryTouchableOpacity';
+import { USERS_PER_PAGE } from '~/components/Trending/constants';
+import { Typography } from '~/components/Typography';
+import { UserFollowList } from '~/components/UserFollowList/UserFollowList';
+import { UserFollowListFallback } from '~/components/UserFollowList/UserFollowListFallback';
+import { OnboardingRecommendedUsersInnerFragment$key } from '~/generated/OnboardingRecommendedUsersInnerFragment.graphql';
+import { OnboardingRecommendedUsersQuery } from '~/generated/OnboardingRecommendedUsersQuery.graphql';
+import { LoginStackNavigatorProp } from '~/navigation/types';
+import { contexts } from '~/shared/analytics/constants';
+import { removeNullValues } from '~/shared/relay/removeNullValues';
+import useFollowAllRecommendedUsers from '~/shared/relay/useFollowAllRecommendedUsers';
+import colors from '~/shared/theme/colors';
+import { noop } from '~/shared/utils/noop';
+
+import { navigateToNotificationUpsellOrHomeScreen } from '../Login/navigateToNotificationUpsellOrHomeScreen';
+
+const HEIGHT_OF_FIXED_ITEMS_ON_SCREEN = 270;
```

Comment:
+ 1 on robin. Adding some comment would be helpful

Reply:
coool, but I think the constants shouldn't come from the components because the heights are specific to this screen.  I added some more constants to explain the breakdown for the `270`.